docs/deploy-cephfs.md

```diff
@@ -44,22 +44,23 @@ that should be resolved in v14.2.3.
-| `--type`                  | _empty_                     | Driver type `[rbd | cephfs]` If the driver type is set to  `rbd` it will act as a `rbd plugin` or if it's set to `cephfs` will act as a `cephfs plugin`                                                                                                                              |
+| `--type`                  | _empty_                     | Driver type `[rbd                                                                                                                                                                                                                                                                    | cephfs]` If the driver type is set to  `rbd` it will act as a `rbd plugin` or if it's set to `cephfs` will act as a `cephfs plugin` |
 | `--instanceid`            | "default"                   | Unique ID distinguishing this instance of Ceph CSI among other instances, when sharing Ceph clusters across CSI instances for provisioning                                                                                                                                           |
 | `--pluginpath`            | "/var/lib/kubelet/plugins/" | The location of cephcsi plugin on host                                                                                                                                                                                                                                               |
 | `--metadatastorage`       | _empty_                     | Points to where older (1.0.0 or older plugin versions) metadata about provisioned volumes are kept, as file or in as k8s configmap (`node` or `k8s_configmap` respectively)                                                                                                          |
 | `--pidlimit`              | _0_                         | Configure the PID limit in cgroups. The container runtime can restrict the number of processes/tasks which can cause problems while provisioning (or deleting) a large number of volumes. A value of `-1` configures the limit to the maximum, `0` does not configure limits at all. |
-| `--metricsport`           | `8080`                      | TCP port for liveness metrics requests                                                                                                                                                                                                                                                  |
+| `--metricsport`           | `8080`                      | TCP port for liveness metrics requests                                                                                                                                                                                                                                               |
```

Comment:
@ShyamsundarR whats the change here for `metricsport` and `enable...` ?  not able to figure out.

Reply:
Merge issue, these need to be removed. Working on it now.

Reply:
Ah! ok, this was not a merge issue after all (which is a relief).

I have seen that whenever we modify YAML tables we are aligning these to the same width. The same was done here post addition of the new option. Although the new option did not change the table width, other added options that did not adjust for the table width (in their commits) were now auto-corrected. IOW, the trailing `|` is now auto-corrected to the right location, which is the change here.

Also, when reviewing this, noticed that the block `[rbd | cephfs]` in option --type was incorrectly aligned due to the table width correction, which is now reverted.